team/team.go

```diff
@@ -0,0 +1,43 @@
+package team
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/astronomer/astro-cli/houston"
+	"github.com/astronomer/astro-cli/pkg/printutil"
+	"github.com/sirupsen/logrus"
+)
+
+// retrieves a team and all of its users if passed optional param
+func Get(args []string, usersEnabled bool, client houston.ClientInterface, out io.Writer) error {
+	teamID := args[0]
+	team, err := client.GetTeam(teamID)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("\n Team Name: %s and team id: %s \n", team.Name, team.ID)
+
+	if usersEnabled {
+		logrus.Debug("retrieving users part of team")
+		fmt.Println("Users part of Team:")
+		users, err := client.GetTeamUsers(teamID)
+		if err != nil {
+			return err
+		}
+		teamUsers := printutil.Table{
```

Comment:
the name of the variable is confusing, I'd change it to `teamUsersTable` or something like this